### DISCUSSION TITLE
Tune Timings

### DISCUSSION THREAD

**Discussion by odflor** [frntc/Sidekick64#86] [General]
Hi!

How I can do to "tune" the timings of the SideKick64?

How I can know what timings is better to my c128 NTSC?

What is better, up of down the timing?

Best Regards

**Comment by frntc** [comment]
What does not work?

You'll find the timings at the last lines of sidekick64.cfg. There is no general rule (and not much experience, as I only have one C128 where I temporarily put in a NTSC VIC). Just to tiny adjustments in +/- 5 steps.
